cosmwasm/cli-utils.js

```diff
@@ -98,12 +99,17 @@ const addChainNameOption = (program) => {
 
 const addAmplifierQueryOptions = (program) => {
     addEnvOption(program);
-
+    addAxelarNodeOption(program);
     addChainNameOption(program);
 };
 
+const addAxelarNodeOption = (program) => {
+    program.addOption(new Option('-u, --node <axelarNode>', 'axelar node url').env('AXELAR_NODE'));
```

Comment:
Use more meaningful name
```suggestion
    program.addOption(new Option('-u, --rpc <axelarRPC>', 'axelar rpc url').env('AXELAR_RPC'));
```